addons/delivery_stock_picking_batch/models/stock_picking.py

```diff
@@ -47,6 +47,8 @@ def _is_auto_batchable(self, picking=None):
         """ Verifies if a picking can be put in a batch with another picking without violating auto_batch constrains.
         """
         res = super()._is_auto_batchable(picking)
+        if not picking:
+            picking = self.env['stock.picking']
```

Comment:
remove `picking` argument and call the method like 
```python
(self | picking)._id_auto_batchable()
```
(in master only :wink:  )